src/example-styles.css

```diff
@@ -24,25 +24,11 @@
     white-space: nowrap;
 }
 
-.multi-select-menuitem--titled:before {
-    display: block;
-    font-weight: bold;
-    content: attr(data-group-title);
-    margin: 0 0 0.25em -20px;
-}
-
-.multi-select-menuitem--titledsr:before {
-    display: block;
+legend {
```

Comment:
The rest of the file only uses class names, probably good to stick with that, and use `.multi-select-legend` here?